test/runtime/configMatters/forall-unordered-opt/PREDIFF

```diff
@@ -0,0 +1 @@
+../../../../util/test/sort-lines-uniq
```

Comment:
Seems fragile WRT to directory moves and since sort-lines-uniq is only one line I'd just inline it

Reply:
That's true, but I prefer not having to rewrite things. I've used this symbolic link pattern in many places.